R/ui_functions.R

```diff
@@ -154,3 +167,140 @@ ui_cp_ki_headcount <- function(country, povline, lkup) {
   return(out)
 
 }
+
+
+#' Provides numbers that will populate the country profile charts
+#'
+#' @param country character: Country code
+#' @param povline numeric: Poverty line
+#' @param pop_units numeric: Units used to express population numbers (default to million)
+#' @param year_range numeric: Range used to subset countries in Poverty MRV chart
+#' @param lkup list: A list of lkup tables
+#'
+#' @return list
+#' @export
+#'
+ui_cp_charts <- function(country = 'AGO', povline = NULL,
+                         pop_units = 1e6, year_range = 2016:2019,
+                         lkup) {
```

Comment:
Shouldn't `year_range` be dynamic?

Reply:
Do you mean like a parameter in the API?